README.md

```diff
@@ -25,14 +26,12 @@ The [desktop tool](https://www.npmjs.com/package/timeld-cli) (called **timeld-cl
 The [Gateway](https://www.npmjs.com/package/timeld-gateway) manages accounts and persists timesheets somewhere safe.
 
 - Register accounts
-- ![coming soon](https://img.shields.io/badge/-coming%20soon-red) Manage projects
-- ![coming soon](https://img.shields.io/badge/-coming%20soon-red) Submit completed timesheets to other management systems
-- ![coming soon](https://img.shields.io/badge/-coming%20soon-red) Import timesheets from other management systems
+- Manage projects
+- Submit completed timesheets to other management systems
```

Comment:
I'd suggest the term "time-tracking systems" in preference to "management systems".